src/mongo/delegates/SearchIndexer.class.php

```diff
@@ -13,6 +13,8 @@
 use Tripod\Mongo\JobGroup;
```

Comment:
Now that you've replaced the `new JobGroup` with `$this->getJobGroup()`, this line can go.